src/main/java/ch/njol/skript/expressions/ExprCommand.java

```diff
@@ -47,48 +45,50 @@
 		"\t\tif the command is not \"exit\":",
 		"\t\t\tmessage \"You're not allowed to use commands during the game\"",
 		"\t\t\tcancel the event"})
-@Since("2.0, INSERT VERSION (support for script commands)")
-@Events("command")
+@Since("2.0, INSERT VERSION (support for script commands and unknown command event)")
+@Events({"command", "unknown command"})
 public class ExprCommand extends SimpleExpression<String> {
 
 	static {
 		Skript.registerExpression(ExprCommand.class, String.class, ExpressionType.SIMPLE,
-				"[the] (full|complete|whole) command",
-				"[the] command [(label|alias)]"
+			"[the] command [(label|alias)]",
+			"[the] (full|complete|whole) command"
```

Comment:
Any reason for this change?

Reply:
I believe change was caused by personal preference will change it back in next push. I want to blame this on being sleep deprived but it was most likely just how I preferred it to be ordered